web/web/api/v1/login.py

```diff
@@ -120,11 +128,16 @@ def process_sign_up():
         refresh_jti = get_jti(encoded_token=refresh_token)
         revoked_store.set(refresh_jti, 'false', JWT_REFRESH_EXPIRES)
 
-        tokens = {
-            'access_token': access_token,
-            'refresh_token': refresh_token
+        login_data = login_schema.dump(new_login)
+
+        results = {
+            'login_data': login_data
         }
```

Comment:
Consolidate the pattern with `create_login_info`:
```python
results = { 'login': login_data }
```

Reply:
done